qwc_services_core/database.py

```diff
@@ -10,22 +10,32 @@ def __init__(self):
         """Constructor"""
         self.engines = {}
 
-    def db_engine(self, conn_str, pool_size=10, max_overflow=20, pool_timeout=30, pool_recycle=300):
+    def db_engine(self, conn_str, pool_size=5, max_overflow=10, pool_timeout=30, pool_recycle=-1):
         """Return engine.
 
         :param str conn_str: DB connection string for SQLAlchemy engine
+        :param int pool_size: Maximum number of possible connections
+        :param int max_overflow: Additional connections beyond pool_size during peak load
+        :param int pool_timeout: Time (in seconds) to wait for a connection to become available
+        :param int pool_recycle: Time (in seconds) after idle connections will be resetted
 
         see https://docs.sqlalchemy.org/en/latest/core/engines.html#postgresql
         """
+
+        db_pool_size = self.db_engine_env('POOL_SIZE', pool_size)
+        db_max_overflow = self.db_engine_env('MAX_OVERFLOW', max_overflow)
+        db_pool_timeout = self.db_engine_env('POOL_TIMEOUT', pool_timeout)
+        db_pool_recycle = self.db_engine_env('POOL_RECYCLE', pool_recycle)
+
```

Comment:
I'd set the default values directly here and remove them from the db_engine args.

Also, `self.db_engine_var` does not return an environment variable value, but invokes `self.db_engine` to return a DB Engine configured with the connections string as configured in the specified environment variable. Use `os.environ.get` here.